vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -176,20 +161,21 @@ kubectl apply -f namespace-resources.yaml
 helm template quickstart ./ > my-clustered-app.yaml
 ```
 
-11. Add environment variables to the `EntandoApp` in the outcome of the Helm template command generated above (`my-clustered-app.yaml`) for the Redis address and Redis password for your deployed Redis instance. The variables to create are:
+11. Add environment variables to `EntandoApp` for your deployed Redis instance per the output of the Helm template command above (`my-clustered-app.yaml`). The variables to create are:
 
 ```
 REDIS_ADDRESS
 ```
 ```
 REDIS_PASSWORD
 ```
+```
+REDIS_ACTIVE
+```
 
-Here is an example. **This is only an example for the EntandoApp and not a complete deployment.** Utilize this as a reference to create your configuration in a complete deployment.
+**The following example pertains to the EntandoApp only, not a complete deployment.** Utilize this as a reference to create your configuration in a complete deployment.
 
->**NOTE**
->
-> This example uses a secret for the `REDIS_PASSWORD` which is recommended. You can also hardcode the password in the YAML for testing. However, creating passwords in clear text in your deployment files is not recommended. Create and use a secret for the password as a best practice.
+>NOTE: This example uses a Secret for the `REDIS_PASSWORD`, which is recommended. You can also hardcode the password in the YAML for testing purposes. However, the use of clear text passwords in deployment files is not recommended.
```

Comment:
Just trying to understand, did something change for 'Create and use a secret for the password as a best practice.'